screwdriver/screwdriver.go

```diff
@@ -186,6 +186,7 @@ type Event struct {
 	ID            int                    `json:"id"`
 	Meta          map[string]interface{} `json:"meta"`
 	ParentEventID int                    `json:"parentEventId"`
+	Creator       map[string]interface{} `json:"creator"`
```

Comment:
Isn't the only type of Creator a string?

```suggestion
	Creator       map[string]string `json:"creator"`
```

Reply:
You are right. I fixed type from interface to string.